13-multiplecomponents-lab/src/02-shoppinglist.jsx

```diff
@@ -18,11 +18,183 @@
  *
  */
 
-function ShoppingItemsCount(props) {
-  return <p>Total items: {props.count}</p>
+import React from 'react';
+
+export default class ShoppingCart extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            items: [],
+            currentItem: {}
+        }
+    }
+
+    addItem = () => {
+        let items = this.state.items;
```

Comment:
If currentItem has quantity 0 - it's logical to pop up some error and not add it to the list.